requirements.txt

```diff
@@ -23,6 +23,6 @@ zope.interface==4.2.0
 # c2corg_common project
 # for development use a local checkout
 # -e ../v6_common
-git+https://github.com/c2corg/v6_common.git@910bb28
+git+https://github.com/c2corg/v6_common.git@a6ce113
```

Comment:
This commit hash is already outdated. https://github.com/c2corg/v6_common/commits/master.
Is your branch rebased?